src/renderers/art/__tests__/ReactART-test.js

```diff
@@ -13,9 +13,6 @@
 
 'use strict';
 
-jest
-  .unmock('ReactART');
```

Comment:
FWIW These files were copied directly from another repo (where they are mostly copied from www). I don't think we can remove that line internally so we'll have to have it in some copies.

It's ok to diverge a tiny bit in this part but if we ever make changes to the actual tests, let's make sure those propagate.